docs/hub/model-repos.md

```diff
@@ -83,14 +83,24 @@ widget:
 - text: "Jens Peter Hansen kommer fra Danmark"
 ```
 
-It is also possible to specify non-text example inputs in the model card metadata. For example, allow users to choose from two sample audio files for automatic speech recognition tasks by:
+It is also possible to specify multiple example inputs & their titles that will be used in the examples picker drop-down of the widget.
```

Comment:
Maybe include a screenshot of picking Sentiment Analysis from the dropdown menu for T0pp, and then match the example YAML snippet below to clearly link the two?

```suggestion
If you have more than one example input, you can name each example input with `example_title`. Users can now select a specific example using the `example_title` from the dropdown menu of the widget.
```